generator/graphql-kotlin-schema-generator/build.gradle.kts

```diff
@@ -2,16 +2,20 @@ description = "Code-only GraphQL schema generation for Kotlin"
 
 val classGraphVersion: String by project
 val graphQLJavaVersion: String by project
+val graphQLJavaDataLoaderVersion: String by project
 val kotlinCoroutinesVersion: String by project
 val rxjavaVersion: String by project
 val junitVersion: String by project
 val slf4jVersion: String by project
 
 dependencies {
-    api("com.graphql-java:graphql-java:$graphQLJavaVersion")
+    api("com.graphql-java:graphql-java:$graphQLJavaVersion") {
+        exclude(group = "com.graphql-java", module = "java-dataloader")
```

Comment:
i.e. if someone pulls over just schema generator (a valid use case since not everyone uses our Spring package) then it will break their server if they don't manually pull in the data loader package